### DISCUSSION TITLE
Verification email during sign up is never sent

### DISCUSSION THREAD

**Discussion by 0xernesto** [WalletConnect/walletconnect-monorepo#4491] [Developer Support] [answered]
<img width="467" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/78889960/740422fd-8cf5-43be-a8f0-597824eb951d">

The signup process via email for cloud.walletconnect.com is awful. The verification email is never sent. The email is not in my spam folder and there is no option to resend the email.

I successfully created an account by connecting my wallet, but I can't add my email to the account because it says my email is already being used.

I also attempted to sign in with my email hoping I could verify my email once logged in, but I'm unable to. I get a message saying my email is unverified.

**Comment by Cali93** [comment, marked as answer]
Hey @0xernesto, thanks for reporting this issue.

We will add a resend verification email button shortly.

**Reply by Cali93** [reply]
Thanks for your patience @0xernesto.

You can go trough the sign up process again with the email you initially tried to sign up with.
Users now have the possibility to resend the confirmation email after signing up, in case you didn't receive it again.

If it's still not working, you can send an email to `cloud at walletconnect.com` with a link to this discussion and your email address so that we can look into your specific case.
